pkg/policies/opa/rego/docker/docker_run/yumupdate.rego

```diff
@@ -0,0 +1,26 @@
+package accurics
+
+{{.prefix}}{{.name}}{{.suffix}}[dockerRun.id] {
+    dockerRun := input.run[_]
+    is_string(dockerRun.config)
+    config := dockerRun.config
+    checkyumUpdate(config)
+}
+
+{{.prefix}}{{.name}}{{.suffix}}[dockerRun.id] {
+    dockerRun := input.run[_]
+    is_array(dockerRun.config)
+    config := dockerRun.config
+    checkyumUpdateArray(config)
+}
+
+checkyumUpdate(config) {
+    contains(config, ["yum update", "yum update-to", "yum upgrade", "yum upgrade-to"][_])
```

Comment:
`config` string may contain multiple commands, you might want to split the string to check all commands

Reply:
okayy, on it